### ISSUE TITLE
Punching when changing cloths

### ISSUE THREAD

**Issue by Smokeprofit** [andristum/dpclothing#25]
Menu was working fine for ages but now when you click on the menu it punches. Still changes cloths. Any way to disable punching when in the wheel?

**Comment by zfbx** [comment]
This actually just started happening for us too, nothing seems out of place in the code, but we did update to a newer dlc?..

**Comment by Smokeprofit** [comment]
Yup same here but now it had fixed itself. I did nothing and now it works.

**Comment by zfbx** [comment]
> Yup same here but now it had fixed itself. I did nothing and now it works.

what artifact are you running?

**Comment by Smokeprofit** [comment]
4380

**Comment by zfbx** [comment]
Yep, that solved it, guess there were some bad artifacts that broke that function for a bit, thanks for the heads up :)

**Comment by Smokeprofit** [comment]
Sweet.
